crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -15416,7 +15416,7 @@ The shadows disperse without effect.
 Die Schatten verziehen sich ohne Wirkung.
 %%%%
 The dungeon shakes, a horrible noise fills the air, and a portal to some otherworldly place is opened!
-Der Dungeon erbebt, ein schreckliches Geräusch erfüllt die Luft, und ein Portal in eine andere Welt wird geöffnet!
+Der Kerker erbebt, ein schreckliches Geräusch erfüllt die Luft, und ein Portal in eine andere Welt wird geöffnet!
```

Comment:
Der Kerker erbebt, ein schreckliches Geräusch erfüllt die Luft und ein Portal in eine andere Welt wird geöffnet!

Reply:
done